onnxruntime/core/framework/tensorprotoutils.cc

```diff
@@ -411,7 +411,12 @@ Status TensorProtoToMLValue(const Env& env, const ORTCHAR_T* tensor_proto_path,
       OrtReleaseStatus(status);
       return Status(common::ONNXRUNTIME, common::FAIL, "initialize preallocated buffer failed");
     }
+    
+#ifdef _WIN32    
+    deleter.f = (void (__stdcall *)(void *) noexcept)DeleteHeapBuffer;
```

Comment:
Instead of forcing converting it to stdcall, we'd better to define the function as stdcall. 
You may use the macro: ORT_API_CALL